integration-tests/utils/integration-helpers.js

```diff
@@ -74,14 +74,29 @@ const withUser = test => async args => {
   }
 }
 
+const addUserToV1Workspace = _.flow(withSignedInPage, withUserToken)(async ({ page, billingProject, v1WorkspaceName, email }) => {
+  await page.evaluate((email, billingProject, v1WorkspaceName) => {
+    return window.Ajax().Workspaces.workspace(billingProject, v1WorkspaceName).updateAcl([{ email, accessLevel: 'WRITER', canCompute: true }])
```

Comment:
nit: It might be worth making a constants file for these access levels, see https://github.com/DataBiosphere/terra-ui/pull/2500/files#diff-ada5c6a1752c4db6c566600346028c476c874042c68d93f0d187b2e0a3a07e04R110 as well. I dislike scattered magic strings with external dependencies. We are trying to do the same for Leo atm...

Reply:
Agreed. I can add that.

Reply:
I might add that as a separate tech debt ticket to avoid scope creep. We have a few of these scattered throughout the code-base as well, and I would want to hit those at the same time: https://broadworkbench.atlassian.net/browse/CA-1360